depends/packages/libmultiprocess.mk

```diff
@@ -24,5 +24,5 @@ define $(package)_build_cmds
 endef
 
 define $(package)_stage_cmds
-  $(MAKE) DESTDIR=$($(package)_staging_dir) install
+  $(MAKE) DESTDIR=$($(package)_staging_dir) install-lib
```

Comment:
In commit "build: Update libmultiprocess library" (2a2dd0a80c61792ad30a7814dff55a9d46391d91)

Note for reviewers: replacing `install` with `install-lib` and `install-bin` here isn't strictly necessary just to update the library, but it takes advantage of recent changes in the new version, and makes the build more minimal.

Reply:
> This could be noted in the commit message, too.

Thanks! The commit message has been updated.